ui/src/pages/plan/rules/RulesContainer.tsx

```diff
@@ -11,29 +11,34 @@ const baseUrl = process.env.REACT_APP_MONEYWISE_BASE_URL;
 // TODO: get from login
 const userid = 'test'
 
-export const RulesContainer = () => {
+export const RulesContainer = ({ onRefresh = () => {} }: { onRefresh?: () => void }) => {
```

Comment:
[note] This will be nicer when we get redux. For now, RulesContainer calls a callback, then sets the current time and passes that to the TransactionsContainer, which triggers the axios hook